importer/importer_utils.py

```diff
@@ -147,22 +148,21 @@ def q_from_wikipedia(language, page_title):
     If the page has no item and is in the article
     namespace, create an item for it.
     """
-    site = pywikibot.Site(language, "wikipedia")
-    repo = site.data_repository()
-    page = pywikibot.Page(site, page_title)
+    wp_site = pywikibot.Site(language, "wikipedia")
+    page = pywikibot.Page(wp_site, page_title)
     summary = "Creating item for {} on {}wp."
     summary = summary.format(page_title, language)
-    wdstuff = wds(repo, edit_summary=summary)
+    wdstuff = wds(create_site_instance("wikidata", "wikidata"), edit_summary=summary)
```

Comment:
I would split this into two statements. `repo = ... `and `wdstuff = wds(repo, ...`